frontend/src/components/static_pages/about.css

```diff
@@ -80,7 +86,9 @@
     font-size: 26px;
   }
   .sub-heading {
```

Comment:
we have `.sub-heading` on the home page as well -> can we put these within `#aboutPageTextRow` so they just happen for the about page